test/MainnetHelper.js

```diff
@@ -31,20 +30,20 @@ const indexTripoolWETH9 = 2;
 const indexCurveOUSDOUSD = 0;
 const indexCurveOUSD3CRV = 1;
 
-function parseUnitsBetweenUSDTAndOUSD(usdtAmount) {
-    let balanceOfUSDTInNatural = ethers.utils.formatUnits(usdtBalance, 6);
-    return ethers.utils.parseUnits(balanceOfUSDTInNatural, 18);
-}
+// function parseUnitsBetweenUSDTAndOUSD (usdtAmount) {
+//     const balanceOfUSDTInNatural = ethers.utils.formatUnits(usdtBalance, 6);
```

Comment:
Should this be commented?

Reply:
It's not used so lint was complaining but I didn't want to delete it since someone may need it later.